docs/docsite/rst/community/maintainers_guidelines.rst

```diff
@@ -119,14 +123,82 @@ Some other general guidelines to encourage contributors:
+* [ ] The changes follow the :ref:`Ansible documentation standards <developing_modules_documenting>` and the :ref:`style_guide`.
+* [ ] The changes follow the :ref:`Development conventions <developing_modules_best_practices>`.
+* [ ] If a new plugin is added, it is one of the `allowed plugin types <https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst#modules-plugins>`_.
+* [ ] Documentation, examples, and return sections use FQCNs for the ``M(..)`` :ref:`format macros <module_documents_linking>` when referring to modules.
+* [ ] Modules and plugins from ansible-core use ``ansible.builtin.`` as an FQCN prefix when mentioned.
+* [ ] When a new option, module, plugin, or return value is added, the corresponding documentation or return sections use ``version_added:`` containing the *collection* version which they will be first released in.
+
+  * This  is typically the next minor release, sometimes the next major release. For example: if 2.7.5 is the current release, the next minor release will be 2.8.0, and the next major release will be 3.0.0).
+* [ ] FQCNs are used for ``extends_documentation_fragment:``, unless the author is referring to doc_fragments from ansible-core.
+* [ ] New features have corresponding examples in the :ref:`examples_block`.
+* [ ] Return values are documented in the :ref:`return_block`.
+
+
+Review tests in the PR
+----------------------
+Review the following if tests are applicable and possible to implement for the changes included in the PR:
+
+
+* [ ] Where applicable, the pull request has :ref:`testing_integration` and :ref:`testing_units`.
+* [ ] All changes are covered. For example, a bug case or a new option separately and in sensible combinations with other options.
+* [ ] Integration tests cover ``check_mode`` if supported.
+* [ ] Integration tests check an actual state of the system, not only what the module reports. For example, if the module changes a file, check that the file was actually changed by using the ``ansible.builtin.stat`` module..
+* [ ] Integration tests check return values, if applicable.
+
+
+Review for merge commits and breaking changes
+---------------------------------------------
+
+* [ ] The pull request does not contain merge commits. See the GitHub warnings at the bottom of the pull request. If merge commits are present, ask the author to rebase the pull request branch.
+* [ ] If the pull request contains breaking changes, ask the author and the collection maintainers if it is really needed and there is a way to not introduce breaking changes. If breaking changes are in present, they must only appear in the next major release and **not** in a minor or patch release. The only exception are breaking changes caused by security fixes that are absolutely necessary to fix the security issue.
```

Comment:
```suggestion
* [ ] If the pull request contains breaking changes, ask the author and the collection maintainers if it is needed, and there is a way not to introduce breaking changes. If breaking changes are present, they must only appear in the next major release and **not** in a minor or patch release. The only exception is breaking changes caused by security fixes that are necessary to fix the security issue.
```

Reply:
LGTM, though i would leave `really` and `absolutely` to emphasize things as unnecessary breaking changes is usually a really bad thing we absolutely try to avoid:)

Reply:
Maybe use MUST/MUST NOT